README.md

```diff
@@ -68,7 +66,7 @@ if (obj1 == obj2) {
 }
 
 // this is better - both wrappers might represent the same object
-if (obj1.sketchObject == obj2.sketchObject) {
+if (obj1.isEqual(obj2.sketchObject)) {
```

Comment:
Isn't that second sketchObject superfluous?

Reply:
ah yes good catch